src/util.js

```diff
@@ -64,3 +64,9 @@ export function isPromise (val) {
 export function assert (condition, msg) {
   if (!condition) throw new Error(`[vuex] ${msg}`)
 }
+
+export function partial (fn, ...bindargs) {
```

Comment:
Let's simplify to:

```js
export function partial (fn, arg) {
  return function () {
    return fn(arg)
  }
}
```

...because spread parameter bloats the output code.

Reply:
simplified to have single argument.